doc/user_guide/postgresql_user_guide.md

```diff
@@ -41,21 +92,109 @@ USER '<user>'
 IDENTIFIED BY '<password>';
 ```
 
+If your setup does not support SSL then simply remove suffix `?ssl=true&sslfactory=org.postgresql.ssl.DefaultJavaSSLFactory`.
+
+
+| Variable | Description |
+|----------|-------------|
+| `<host>` | Hostname or ip address of the machine hosting you PostgreSQL database. |
+| `<port>` | Port of the PostgreSQL database. Default port is `5432`, see also [Finding Out the Port of a PostgreSQL Database Installation](#finding-out-the-port-of-a-postgresql-database-installation). |
+| `<schema name>` | Name of the database schema you want to use in the PostgreSQL database. |
+
+
+## Finding Out the Port of a PostgreSQL Database Installation
+
+PostgreSQL default port is `5432`.<br />
+To inquire port in other cases use
+
+```shell
+function hfield () { head -1 | sed -e 's/  */\t/g' | cut -f $1 ; }
+SENDQ=$(ss -tl | grep postgresql | hfield 3)
+ss -tln | grep $SENDQ | hfield 4
+```
+
+## Making PostgreSQL Service Listen to External Connections
+
+See
+* https://www.bigbinary.com/blog/configure-postgresql-to-allow-remote-connection
+* https://dba.stackexchange.com/questions/83984/
+
+Use `sudo vi` to add the following line to file `/etc/postgresql/10/main/postgresql.conf`:
+```
+listen_addresses = '*'
+```
+
+Use `sudo vi`to add the following line to file `/etc/postgresql/10/main/pg_hba.conf`:
+```
+# TYPE DATABASE USER CIDR-ADDRESS  METHOD
+host  all  all 0.0.0.0/0 md5
+```
+
 ## Creating a Virtual Schema
 
-Below you see how a PostreSQL Virtual Schema is created.
+Use the following SQL command in Exasol database to create a Postgres Virtual Schema:
 
 ```sql
 CREATE VIRTUAL SCHEMA <virtual schema name>
-	USING ADAPTER.JDBC_ADAPTER 
+	USING ADAPTER.JDBC_ADAPTER
 	WITH
 	CATALOG_NAME = '<catalog name>'
 	SCHEMA_NAME = '<schema name>'
 	CONNECTION_NAME = 'POSTGRESQL_CONNECTION'
+	DEBUG_ADDRESS = '172.17.0.1:<port>'
+	LOG_LEVEL = 'ALL'
 	;
 ```
 
-## Postgres Identifiers
+| Variable | Description |
+|----------|-------------|
+| `<virtual schema name>` | Name of the virtual schema you want to use. |
+| `<catalog name>` | Name of the catalog, usally equivalent to the name of the PostgreSQL database. |
+| `<schema name>` | Name of the database schema you want to use in the PostgreSQL database. |
+| `<docker host>` | ip address of the docker container running the Exasol database. |
+| `<port>`        | Port you want to use for logging debug information. |
+
+## Display Log Output of Virtual Schema
+
+When creating a virtual schema you can optionally configure it to write log output to a specific host and port.<br />
+In order to display log output you can use the unix command `nc`.
+The following command starts `nc` listening (`-l`), kept alive (`-k`) on port (`-p`)  `<port>`:
+```
+nc -lkp <port>
+```
+
+| Variable | Description |
+|----------|-------------|
+| `<port>` | Port `nc` should listen to. |
+
```

Comment:
Too much replication. Use links! Otherwise you will have to maintain that in all VS. I know this is useful from a customer perspective, but it is a maintenance nightmare.

Reply:
OK, I agree, thanks.